.storybook/Checkbox.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import styled from 'styled-components'
+import { storiesOf, action } from '@storybook/react'
+import { Checkbox, theme } from '../src'
+
+storiesOf('Checkbox', module).add('Checkbox states', () => (
+  <div>
+    <section>
+      <h4>Checkbox</h4>
```

Comment:
Trivial, but it would be a good, consistent idea if your story also uses components from design system, instead of `div`, `section`, `h4. `Box` and `Heading` are available. `section` tag can be achieve with `. withComponent` from styled-components API.